src/overview/components/Filters.css

```diff
@@ -79,3 +79,10 @@ input[type='checkbox'] {
         border-radius: 1px;
     }
 }
+
+.clear {
```

Comment:
Fixed the alignment but confused with the design of the button, shall I keep it a text as it was before?

Reply:
Yes, can just set `border: 0;`. The `position: absolute;` shouldn't be necessary here also. It should fall into place better if you change the `flex-flow` to `row` in `.filtersMain` class (places flex box children horizontally rather than vertically; no idea the reasoning behind current setting) + remove the auto margin on `.bookmarks` class. See what that looks like